repose-aggregator/components/filters/ip-classification/src/main/resources/META-INF/schema/examples/ip-classification.cfg.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+  Repose
+  _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+  Copyright (C) 2010 - 2015 Rackspace US, Inc.
+  _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+  -->
+
+<ip-classification xmlns="http://docs.openrepose.org/repose/ip-classification/v1.0">
+    <!--
+     specify what header to put the group name in. This is the default. It will always add an additional header, not
+     clobbering the existing value.
+    -->
+    <group-header-name quality="0.4"/>
+
+    <!-- then create 0..n classifications. They are ordered, so the first match wins.
+    A default catch all is not supplied. If you want to have it always label, define one.
+    -->
+    <classifications>
+        <!-- Classifications follow the CIDR rules: https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing
+        You can specify ipv4-cidr and/or ipv6-cidr. If either matches, the label is applied to the specified header
+        and matching stops. If neither matches, nothing happens.
+        labels must be unique.
+        You may specify multiple cidr notations space separated in either IPv4 or IPv6.
+        -->
+        <classification label="sample-group" ipv4-cidr="192.168.1.0/24 192.168.0.1/32"/>
+        <classification label="sample-ipv6-group" ipv6-cidr="2001:db8::/48"/>
+        <classification label="bolth-group" ipv4-cidr="10.10.220.0/24" ipv6-cidr="2001:1938:80:bc::1/64"/>
```

Comment:
It's just a label :P It could be anything. And "both" sometimes has "L"s in it. I'm trying to be inclusive.